Ports/poppler/package.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+export VERSION_DATA='0.4.12'
+
+port='poppler'
+version='23.07.0'
+files="https://poppler.freedesktop.org/poppler-${version}.tar.xz poppler-${version}.tar.xz f29b4b4bf47572611176454c8f21506d71d27eca5011a39aa44038b30b957db0
+https://poppler.freedesktop.org/poppler-data-${VERSION_DATA-data}.tar.gz poppler-data-${VERSION_DATA}.tar.gz c835b640a40ce357e1b83666aabd95edffa24ddddd49b8daff63adb851cdab74"
```

Comment:
Why default `${VERSION_DATA}` to `data` if it's always set above?

Reply:
That is absolutely a copy&paste error, that should not be there.